eng/common/scripts/Update-ChangeLog.ps1

```diff
@@ -74,21 +75,21 @@ if ($ChangeLogEntries.Contains($Version))
 {
     if ($ChangeLogEntries[$Version].ReleaseStatus -eq $ReleaseStatus)
     {
-        LogWarning "Version [$Version] is already present in change log with specificed ReleaseStatus [$ReleaseStatus]. No Change made."
+        LogDebug "Version [$Version] is already present in change log with specificed ReleaseStatus [$ReleaseStatus]. No Change made."
```

Comment:
These warnings in some common scenarios are outputting warnings which aren't interesting so turning these into debug  logging instead of warning.